tests/unit/amo/components/TestAddon.js

```diff
@@ -866,34 +867,47 @@ describe(__filename, () => {
   });
 
   it('renders a static theme preview as an image', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/54321.png';
+    const newPreview = [
```

Comment:
You don't really need this `newPreview` variable - you could just inline this into the call to `shallowRender`.